source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,340 @@
+    ```
+
+    The information needed to install and enable pathauto and its dependencies is now committed to your GitHub repository. The modules sources themselves, however, will not be part of this commit.
+
+## Create a Custom Theme
+
+A custom theme is the most convenient way to add css styles or alter the markup produced by modules. Most Drupal sites will have a custom theme to differentiate the appearance of the site. For comprehensive documentation on how to create themes for Drupal 8, see [Theming Drupal 8](https://www.drupal.org/docs/8/theming) on drupal.org.
+
+A simple theme may be created quite easily using Drupal Console. The example below creates a subtheme of [Bartik](https://www.drupal.org/project/bartik) to allow for simple css changes for demonstration purposes. Note that usually, themes will subclass either `classy` or `stable`, which provide a blank slate to start from.
+
+1.  Run the `generate:theme` command as shown below to start the process of creating a subtheme.
+
+    ```
+    terminus drupal $SITE.$ENV -- generate:theme
+    ```
+    Drupal Console will ask a series of questions about your theme. For many of them you can use the default value by just hitting **Enter**. Answer these quesions as follows:
+
+    - Theme: Amazing Theme
+    - Base theme: bartik
+    - Enter 'no' to generate theme regions and theme breakpoints
+
+    Type `yes` when asked to confirm generation of the theme. Once you do this, the files for your new theme will be written to the directory `code/web/themes/custom/amazing_theme`. If you gave your theme a different name, replace `amazing_theme` with the appropriate name for your theme.
+
+2.  Use an SFTP client to access the generated theme files in [Pantheon's on-server development mode](https://pantheon.io/docs/sftp/). Find the file `code/web/core/theme/bartik/bartik.info.yml`, and open it in an editor. Copy the regions section starting on line 28, and paste it into the file `code/web/themes/custom/amazing_theme/amazing_theme.info.yml`. Optionally, you might also want to copy the file `code/web/core/theme/bartik/logo.svg` into your theme folder.
+
+3.  In your new theme's directory, create a file named `amazing_theme.libraries.yml`, and put the following contents in it:
+
+    ```
+    global-styling:
+      version: VERSION
+      css:
+        theme:
+          css/main.css: {}
+    ```
+4.  Create a folder named `css` in your theme directory and create a file named `main.css` inside it. To test that your theme is working, add some very obvious styling such as the red border around the content region shown in the example below:
+
+    ```
+    #content {
+      border: 4px solid red;
+    }
+    ```
+
+    Remember to upload your updated files back to the server over SFTP.
+
+5.  Use Terminus with Drupal Console and Drush to active your new theme and rebuild the cache:
+
+    ```
+    terminus drupal $SITE.$ENV theme:install -- set-default amazing_theme
```

Comment:
Typo still exists in the `theme:install` example above. It should be either:
```
terminus drupal $SITE.$ENV theme:install -- --set-default amazing_theme
```
Or perhaps:
```
terminus drupal $SITE.$ENV theme:install -- amazing_theme --set-default
```
The later might be clearer, since it doesn't have four `--` in a row. On the flip side, though, that extra space between the `--` and the theme name is extremely critical. Might be confusing for folks who accidentally leave it out. Putting the `--set-default` next to the `--` might be a little clearer in that respect, even if it is harder to read.